### PR TITLE
Extended "SubHeaderLabel" for TVShows, Movies, PVR and Weather

### PR DESCRIPTION
**Pull request by semool**
Kodi_20210401_205444
Kodi_20210401_205500
Kodi_20210401_205418

### CODE REVIEW

File: 1080i/Includes_Furniture.xml [hunk 1]
```diff
@@ -2929,6 +2929,15 @@
...
         <value condition="!String.IsEmpty(Container(9500).NumItems) + !Window.IsVisible(home)">[COLOR=$VAR[ColorHighlight]]$INFO[Container(9500).NumItems,,[/COLOR] $LOCALIZE[31101]]</value>
+
+        <value condition="String.IsEqual(Container(300).ListItem.Property(defaultID),tvshows)">[COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Episodes.Count)][/COLOR] $LOCALIZE[20360]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(Home).Property(episodes.UnWatched)][/COLOR] $LOCALIZE[31215]</value>
+        <value condition="String.IsEqual(Container(300).ListItem.Property(defaultID),movies)">[COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Movies.Count)][/COLOR] $LOCALIZE[20342]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(Home).Property(Movies.UnWatched)][/COLOR] $LOCALIZE[31215]</value>
+        <value condition="String.IsEqual(Container(300).ListItem.Property(defaultID),livetv)">$INFO[Pvr.BackendName]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Pvr.BackendChannels][/COLOR] $LOCALIZE[19019]</value>
```
Comment (beatmasterRS) [inline 1080i/Includes_Furniture.xml:2935]: Isn't this condition always true?

File: 1080i/Includes_Furniture.xml [hunk 2]
```diff
@@ -2929,6 +2929,15 @@
...
+        <value condition="String.IsEqual(Container(300).ListItem.Property(defaultID),livetv) + String.IsEmpty(Pvr.BackendChannels)">$INFO[Pvr.BackendName]</value>
```
Comment (beatmasterRS) [inline 1080i/Includes_Furniture.xml:2936]: And isn't this condition always false?

**Review by beatmasterRS — CHANGES_REQUESTED** [review]
Looks good :)

Please can you add the TVShows.Count property to the tv show section? i think its related. So its looks like "111 TV Shows  •  111 Episodes  •  111 unwatched"?

And take a look at the lines, i've commented :)

**Comment by semool** [comment]
> Looks good :)
> 
> Please can you add the TVShows.Count property to the tv show section? i think its related. So its looks like "111 TV Shows • 111 Episodes • 111 unwatched"?

Yes i would do this in the beginning but it used to much space and overlaps with the Clearart in the top middle when enabled. And the total Count of Shows is displayed when you enter the Series Section.

File: 1080i/Includes_Furniture.xml (hunk 1 above)
Comment (semool) [reply 1080i/Includes_Furniture.xml:2935]: No its true when you select the ID (tvshows, movies....) in the MainMenu

**Comment by beatmasterRS** [comment]
Thanks for your reply. 

Line 2935 is true, when `String.IsEqual(Container(300).ListItem.Property(defaultID),livetv)`, but its also true, when `String.IsEmpty(Pvr.BackendChannels)`

So if `String.IsEmpty(Pvr.BackendChannels)`, i get "$INFO[Pvr.BackendName]  •  " which doesnt look good. 

Line 2936 will can be true, or a  i wrong? Is could not test it.

**Comment by semool** [comment]
As far as I know, these lines are processed one after the other. Which means yes 2935 is true when String.IsEmpty(Pvr.BackendChannels) but then it will replaced with line 2936 which is only true with String.IsEmpty(Pvr.BackendChannels).
Or im wrong?

**Pull request merged**